homeassistant/components/broadlink/climate.py

```diff
@@ -0,0 +1,125 @@
+"""
+Support for Chinese wifi thermostats (Floureon, Beok, Beca Energy).
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.broadlink/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA, STATE_OFF
+from homeassistant.components.climate.const import (
+    SUPPORT_TARGET_TEMPERATURE,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_MAC,
+    PRECISION_HALVES,
+    TEMP_CELSIUS,
+    CONF_HOST,
+)
+import homeassistant.helpers.config_validation as cv
+
+DEFAULT_NAME = "Broadlink Thermostat"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler